Frameworks/Foundation/NSCache.mm

```diff
@@ -286,7 +286,7 @@ - (void)_evictEntriesOverCharge {
 
     auto rit = _cacheEntries.end();
     do {
-        _NSCacheEntry* entry = (id)*--rit;
+        _NSCacheEntry* entry = (id) * --rit;
```

Comment:
Many of the other files can be reverted as well (in Frameworks\Include)

---
In reply to: [70353130](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70353130) [](ancestors = 70353130,70347524)